rero_ils/modules/items/api_views.py

```diff
@@ -82,7 +82,7 @@ def decorated_view(*args, **kwargs):
         except NotFound as error:
             raise(error)
         except Exception as error:
-            # raise(error)
+            raise(error)
```

Comment:
If I recall, you need to keep this line commented

Reply:
done